src/foam/u2/view/BadgeView.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'BadgeView',
+  extends: 'foam.u2.View',
+
+  css: `
+    ^{
+      border-radius: 11.2px
+      font-family: /*%FONT1%*/ Roboto, 'Helvetica Neue', Helvetica, Arial, sans-serif;
+      font-size: 10px;
+      font-style: normal;
+      font-stretch: normal;
+      font-weight: 500;
+      height: 24px;
+      line-height: 24px;
+      letter-spacing: normal;
+      padding: 0 8px;
+      text-align: center;
+      width: 79px;
+    }
+   
+  `,
+
+  documentation: `Creates badges with rounded/squared sides based on display context`,
+
+  properties: [
+    {
+        class: 'String',
+        name: 'label'
```

Comment:
Can we just use 'data' instead of 'label', to be consistent with all other Views? In which case, you can just remove this property since 'View' already has a data property.